scripts/release.sh

```diff
@@ -128,6 +128,8 @@ function advance_admiral() {
     for project in ${OPERATOR_CONSUMES[*]}; do
         write_component
     done
+    # TODO: move this part to advance_projects
```

Comment:
```suggestion

```

What I meant was the function names can be changed to make them easier to follow, this specific line is in the right place

Reply:
I'm not sure I understand...  I think I lost the scope.  Should I revert anything? can you please explain again?